packages/nitro-protocol/src/contract/outcome.ts

```diff
@@ -140,7 +140,7 @@ export type Outcome = AssetOutcome[];
 
 export function hashOutcome(outcome: Outcome): Bytes32 {
   const encodedOutcome = encodeOutcome(outcome);
-  return utils.keccak256(utils.defaultAbiCoder.encode(['bytes'], [encodedOutcome]));
```

Comment:
This is the change we would need to replicate in `native-utils`.

Reply:
I'll take a look today.

Reply:
https://github.com/statechannels/native-utils/pull/9